lms/static/js/student_account/views/PasswordResetView.js

```diff
@@ -1,48 +1,45 @@
-var edx = edx || {};
-
-(function($, gettext) {
+;(function (define) {
     'use strict';
+    define(['jquery', 'js/student_account/views/FormView'], function($, FormView) {
 
-    edx.student = edx.student || {};
-    edx.student.account = edx.student.account || {};
-
-    edx.student.account.PasswordResetView = edx.student.account.FormView.extend({
-        el: '#password-reset-form',
+        var PasswordResetView = FormView.extend({
```

Comment:
Instead of assigning the view to a variable and returning that you can just return the view, i.e.
```
return FormView.extend({
```